src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/SubjectAccessRequestController.kt

```diff
@@ -27,6 +29,7 @@ import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.publicApi.S
 @RequestMapping("/public", produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
Just noticed this, but it shouldnt be on the /public path

Reply:
private?